test/tasks/demo.test.js

```diff
@@ -15,7 +14,7 @@ const oTestPath = 'test/fixtures/o-test';
 const pathSuffix = '-demo';
 const demoTestPath = path.resolve(obtPath, oTestPath + pathSuffix);
 
-describe('Demo task', function() {
+describe('Demo task', () => {
```

Comment:
Generally better to do ES6 syntax refactoring in a separate branch/PR so it doesn't get mixed in